bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html

```diff
@@ -0,0 +1,62 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_whatsnew_105_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main mzp-l-content mzp-t-content-lg">
+  <h2 class="wnp-main-title">How to talk to your kids about online safety</h2>
+
+  <img src="{{ static('img/firefox/whatsnew/whatsnew105/tech-talk.png') }}" width="720" height="405" alt="The Tech Talk" class="wnp-main-img">
+
+  <p class="wnp-main-tagline">Firefox can help make awkward conversations less awkward. Check out The Tech Talk to get pointers and download the Super-Official Family Tech Agreement.</p>
+
+  <p class="wnp-main-cta">
+    <a class="mzp-c-button mzp-t-product mzp-t-lg" href="{{ url('firefox.family.index') }}" data-cta-text="Start the conversation" data-cta-type="button">
+      Start the conversation
+    </a>
+  </p>
+</section>
+
+<section class="wnp-content-extra">
+  <div class="mzp-l-content mzp-t-content-lg">
+
+    <h3 class="mzp-c-section-heading">Some topics you’ll cover:</h3>
+
+    <div class="mzp-l-columns mzp-t-columns-three mzp-t-picto-center">
+      <div class="mzp-c-picto">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="" alt="216" class="mzp-c-picto-image">
```

Comment:
NIt: looks like you need to swap the `height` and `alt` values around on these three images:
```suggestion
        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="216" alt="" class="mzp-c-picto-image">
```

Reply:
🤦